docs/UserGuide.adoc

```diff
@@ -341,7 +342,100 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-== Features (Coming in v2.0)
+== Features(Coming in v2.0)
+
+=== Add a relationship
+
+Adds a relationship between two persons. +
+Format: `addRelationship INDEXOFSOURCE INDEXOFDESTINATION DIRECTION`
+
+****
+* Add a relationship between the person with `INDEXOFSOURCE` and `INDEXOFDESTINATION`, +
+with the direction of the relationship specified.
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
+* `INDEXOFDESTINATION` refers to the index of the person to whom the relationship is directed.
+* `DIRECTION` referring to the direction of this relationship, can only be String input of `directed` or `undirected`.
```

Comment:
"which refers to"

"....can only be `directed` or `undirected`" (this is a user guide)